lib/webpacker/manifest.rb

```diff
@@ -23,11 +23,21 @@ def refresh
   end
 
   def lookup(name)
-    compile if compiling?
+    initialize!
     find name
```

Comment:
imo find(name) is better

Reply:
If you're referring to the `lookup()` method?  I believe that was pre-existing.